python/lsst/ap/verify/pipeline_driver.py

```diff
@@ -202,6 +202,23 @@ def runApPipe(metricsJob, workspace, parsedCmdLine):
 
     dataId = _parseDataId(parsedCmdLine.dataId)
     processes = parsedCmdLine.processes
+    
+    # borrowed from validate_drp
+    def extract_instrument_from_butler(butler):
+        """Extract the last part of the mapper name from a Butler repo.
+        'lsst.obs.lsstSim.lsstSimMapper.LsstSimMapper' -> 'LSSTSIM'
+        'lsst.obs.cfht.megacamMapper.MegacamMapper' -> 'CFHT'
+        'lsst.obs.decam.decamMapper.DecamMapper' -> 'DECAM'
+        'lsst.obs.hsc.hscMapper.HscMapper' -> 'HSC'
+        """
+        camera = butler.get('camera')
+        instrument = camera.getName()
+        return instrument.upper()
+    
+    #  Insert job metadata
+    metricsJob.meta.update({'instrument': extract_instrument_from_butler(workspace.workButler),
+                            'filter_name': dataId['filter']})
```

Comment:
I think requires is too strong.  There is an assumption inherited from the `validate_drp` work, but the verify framework certainly doesn't require this.  It's useful metadata to have.  @afausti can say more, but I think the upcoming visualization framework will make this assumption go away.

Reply:
Ok, so until the old framework goes away, how should we handle metrics that can't be meaningfully associated with a filter?

Reply:
To bring this to a close, we have decided to deprecate the old framework to avoid this problem.  This is not an issue since there are no historical AP metrics in the old framework and would have only been populated for symmetry with DRP going forward.